ffshot.c

```diff
@@ -26,6 +26,9 @@
 #define htobe16 htons
 #endif
 
+#define IC xcb_get_image(con, XCB_IMAGE_FORMAT_Z_PIXMAP, wid, 0, 0, width, height, ~0)
```

Comment:
I don't like this macro at all.

It only gets used once, is not an obvious function call and makes the code harder to read, please revert this change. `CD(ch)` would be a fine macro.